core/rawdb/accessors_chain.go

```diff
@@ -695,33 +722,33 @@ func DeletePhCache(db ethdb.KeyValueWriter) {
 	}
 }
 
-// ReadCurrentPendingHeaderHash retreive's the heads hashes of the blockchain.
-func ReadCurrentPendingHeaderHash(db ethdb.Reader) common.Hash {
+// ReadBestPhKeyObject retreive's the bestPhKeyObject of the blockchain
+func ReadBestPhKeyObject(db ethdb.Reader) types.BestPhKeyObject {
 	data, _ := db.Get(phHeadKey)
 	// get the ph cache keys.
 	if len(data) == 0 {
-		return common.Hash{}
+		return types.BestPhKeyObject{}
 	}
-	hash := common.Hash{}
-	if err := rlp.DecodeBytes(data, &hash); err != nil {
-		return common.Hash{}
+	bestPhKey := new(types.BestPhKeyObject)
+	if err := rlp.DecodeBytes(data, bestPhKey); err != nil {
+		return types.BestPhKeyObject{}
 	}
-	return hash
+	return *bestPhKey
 }
 
-// WriteCurrentPendingHeaderHash writes the heads hashes of the blockchain.
-func WriteCurrentPendingHeaderHash(db ethdb.KeyValueWriter, hash common.Hash) {
-	data, err := rlp.EncodeToBytes(hash)
+// ReadBestPhKeyObject writes the bestPhKey of the blockchain
```

Comment:
Confusing comment. Probably mean `WriteBestPhKeyObject` instead.

Reply:
No need now since we use it as bestPhKey